backend/constants/Carousels.ts

```diff
@@ -0,0 +1,19 @@
+let CAROUSEL_DATA = [
+  {
+    image: "https://www.kermanig.com/cdn/shop/files/Sarkis.jpg",
+  },
+  {
+    image:
+      "https://www.kermanig.com/cdn/shop/files/Orange_and_Green_Abstract_Shapes_Free_Shipping_Nationwide_Banner_09ab6816-8a57-4217-9d47-990a6742ed68.png",
+  },
+  {
+    image: "https://www.kermanig.com/cdn/shop/files/Monthly_promotions_4.jpg",
+  },
+];
+
+export const GetCAROUSEL_DATA = () => {
+  return CAROUSEL_DATA;
+};
+export const SetCAROUSEL_DATA = (val: Array<{ image: string }>) => {
+  return (CAROUSEL_DATA = val);
+};
```

Comment:
```suggestion
export const getCarouselData = () => {
  return CAROUSEL_DATA;
};
export const setCarouselData = (val: Array<{ image: string }>) => {
  return (CAROUSEL_DATA = val);
};
```
All JavaScript functions need to start with lowercase.